bin/stacks/api-stack.ts

```diff
@@ -316,7 +316,18 @@ export class APIStack extends cdk.Stack {
       },
     });
     const enabled = switchResource.addResource('enabled');
-    enabled.addMethod('GET', switchLambdaIntegration);
+
+    /* add auth key */
+    const apiAuthzKey = api.addApiKey('AuthzKey');
```

Comment:
where is this value being generated / fetched? I was envisioning a pattern more similar to how we do the GPA internal key for URA

Reply:
key is generated right here! We store it in pipeline secret manager and URA reads it and include it in x-api-key header when calling `enabled`

Reply:
oh cool, is Authz a typo or related to this?